rpcs3/Emu/RSX/overlays.cpp

```diff
@@ -2,16 +2,23 @@
 #include "overlays.h"
 #include "GSRender.h"
 
+#include "Emu/Cell/SPUThread.h"
+#include "Emu/Cell/RawSPUThread.h"
+#include "Emu/Cell/PPUThread.h"
+#include "Utilities/sysinfo.h"
+#include "Utilities/StrFmt.h"
+#include "Utilities/StrUtil.h"
+
 namespace rsx
 {
 	namespace overlays
 	{
-		//Singleton instance declaration
+		//	Singleton instance declaration
```

Comment:
Why? Also happens below

Reply:
it happens when you use clang format in VS.

``//Comment``
becomes
``//->Comment``

``// Comment``
is fine though

Reply:
yea I've run clang-format on everything
if formatting is questionable in places, it's not my fault :^